addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -77,7 +77,7 @@ export class AnalyticDistribution extends Component {
 
         this.selectCreateIsOpen = false;
```

Comment:
one2many autocompletes nope: you change Many2XAutocomplete, that means many more

Reply:
how many records nope: it is not there if there is no records